### ISSUE TITLE
Wow we made the same lib

### ISSUE THREAD

**Issue by libetl** [rockswang/java-curl#2]
I think we can reuse our work to improve our respective libraries. Don't you mind if I look ?
Thank you
(https://github.com/libetl/curl)

**Comment by rockswang** [comment]
you are welcome :), /shake hands

**Comment by Darkensses** [comment]
Yes please. The @rockswang 's lib has the json response and this is so important for automation proposses. I think the same like @libetl

**Comment by Darkensses** [comment]
Yes please. The @rockswang 's lib has the json response and this is so important for automation purposes. I think the same like @libetl

**Comment by shamiwillms** [comment]
Is there any way to show a progress dialog (in Java) using either of these libraries?

**Comment by rockswang** [comment]
@shamiwillms not possible in my lib

**Comment by shamiwillms** [comment]
okay...thanks a lot for the quick response.  You have a great library here.
